### PR TITLE
Adjust some error text and avoid some useless writes to log

### PR DESCRIPTION
**Pull request by martinlindhe**
Follow-up to DEV-1668

### CODE REVIEW

File: handler.go [hunk 1]
```diff
@@ -72,11 +71,13 @@ func (ca *Cagent) Run(outputFile *os.File, interrupt chan struct{}) {
 				// for error codes 5xx, wait for configured amount of time and try again
 				retries++
 				if retries > ca.Config.OnHTTP5xxRetries {
-					log.Errorf("hub connection error. giving up")
+					log.Errorf("Run: hub connection error. giving up")
```
Comment (m-terel) [inline handler.go:74]: if no args please use `log.Error` instead.

File: handler.go [hunk 2]
```diff
@@ -72,11 +71,13 @@ func (ca *Cagent) Run(outputFile *os.File, interrupt chan struct{}) {
...
 					return
 				}
-				log.Infof("hub connection error %d/%d, retrying in %v s", retries, ca.Config.OnHTTP5xxRetries, ca.Config.OnHTTP5xxRetryInterval)
+				log.Infof("Run: hub connection error %d/%d, retrying in %v s", retries, ca.Config.OnHTTP5xxRetries, ca.Config.OnHTTP5xxRetryInterval)
```
Comment (m-terel) [inline handler.go:77]: Maybe would be better to use `Warn` instead of `Info`?

File: handler.go [hunk 3]
```diff
@@ -287,21 +288,21 @@ func (ca *Cagent) RunHeartbeat(interrupt chan struct{}) {
...
 				time.Sleep(10 * time.Second)
 				continue
-			}
-			if err == ErrHubServerError {
+			} else if err == ErrHubServerError {
 				// for error codes 5xx, wait for configured amount of time and try again
 				retries++
 				if retries > ca.Config.OnHTTP5xxRetries {
-					log.Errorf("hub connection error. giving up")
+					log.Errorf("RunHeartbeat: hub connection error. giving up")
```
Comment (m-terel) [inline handler.go:299]: if no args please use `log.Error` instead.

File: handler.go [hunk 4]
```diff
@@ -287,21 +288,21 @@ func (ca *Cagent) RunHeartbeat(interrupt chan struct{}) {
...
 					return
 				}
-				log.Infof("hub connection error %d/%d, retrying in %v s", retries, ca.Config.OnHTTP5xxRetries, ca.Config.OnHTTP5xxRetryInterval)
+				log.Infof("RunHeartbeat: hub connection error %d/%d, retrying in %v s", retries, ca.Config.OnHTTP5xxRetries, ca.Config.OnHTTP5xxRetryInterval)
```
Comment (m-terel) [inline handler.go:302]: The same, maybe `Warn`?

File: handler.go (hunk 2 above)
Comment (martinlindhe) [reply handler.go:77]: From issue description:
> Failed connections are logged with severity info. Only if all retries have failed, an error is logged.

File: handler.go (hunk 1 above)
Comment (martinlindhe) [reply handler.go:74]: Done

File: handler.go (hunk 3 above)
Comment (martinlindhe) [reply handler.go:299]: Done

File: handler.go (hunk 4 above)
Comment (martinlindhe) [reply handler.go:302]: See other comment

**Review by m-terel — APPROVED** [review]

**Pull request merged**